integration-tests/utils/integration-utils.js

```diff
@@ -29,37 +29,39 @@ const findInGrid = async (page, text) => {
   return await page.waitForXPath(`//*[@role="grid"][contains(normalize-space(.),"${text}")]`)
 }
 
-const exactlyFindClickable = (page, text) => {
-  return page.waitForXPath(`(//a | //*[@role="button"] | //button)[normalize-space(.)="${text}"]`)
+const clickable = ({ text, textContains }) => {
```

Comment:
TOL: Currently, `textContains` is the more frequently used property. I think the more frequently used property should be the most concise one.

There are two ways to accomplish this:
1. Rename these to something like `{ text, exactText }`.
2. Change the current tests to use `text` as it's currently defined (updating the search text as needed).

This is an opportunity to be opinionated about which form is preferred.

Side note: I recognize that the existing naming is consistent with the `contains` xpath function. We could also choose to keep the current naming for consistency with xpath vocabulary. IMO (70%)*, these functions exist as an abstraction on top of xpath so consistency with xpath vocabulary is less important than clarity/fluency of these functions' APIs.

\* Trying out a notation to describe how strongly I hold this opinion.

Reply:
I'm on the fence here. The main reason `textContains` is more used at the moment is because that's what I happened to implement initially. But there are cases where one or the other would be preferable. And both formulations here seem decent. 🤷‍♂

Reply:
I thought about this a little more, and given that substring matching is a more complex operation than regular equality, I like that it has the slightly more interesting name.